.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "16"
+  - "20"
```

Comment:
Travis is still erroring, please try copying (and adapting) the dist & version from here: https://github.com/okTurtles/group-income/blob/master/.travis.yml